test/factories/item.factory.ts

```diff
@@ -0,0 +1,38 @@
+import { faker } from '@faker-js/faker';
+
+import { CCLicenseAdaptions, ItemType } from '@graasp/sdk';
+
+import { Item } from '../../src/services/item/entities/Item';
+
+/**
+ * This factory does not guarantee valid items given their type. But they are acceptable when seed into the db.
+ * Create by default a folder
+ * @param item partial properties
+ * @returns complete item valid to be fed into the database
+ */
+export const ItemFactory = (item: Partial<Item>) => ({
+  type: ItemType.FOLDER,
+  order: null,
+  name: faker.word.words(2),
+  description: item.description ?? faker.lorem.text(),
```

Comment:
Why use a nullish here when you spread the item below ? It is not sufficient to spread below ?